brave-core/src/test/java/com/github/kristofa/brave/LocalTracerTest.java

```diff
@@ -82,6 +91,38 @@ public void startNewSpan() {
     }
 
     /**
+     * When a local span is started without a timestamp, microseconds and a tick are recorded for
```

Comment:
I'd not copy/paste javadoc for this method as its intent is testing something different. For example, its focus isn't microseconds, rather local parent.